metabrainz/templates/index/code-of-conduct.html

```diff
@@ -25,22 +25,22 @@ <h1 class="page-title">{{ _('Code of Conduct') }}</h1>
       agree with them.
     </li>
     <li>
-      MetaBrainz projects have global communities. Be mindful and respectful of different languages, habits and cultures.
+      MetaBrainz projects have global communities. Be mindful and respectful of different languages, habits, and cultures.
     </li>
     <li>
-      Share knowledge freely. New members should not be embarrassed for ignorance.
+      Share knowledge freely. New members should not be embarrassed about ignorance.
     </li>
     <li>
       Respectfully suggest other options. Not everyone wants to reconfigure their directories, change their OS or switch players and formats.
     </li>
     <li>
       Try not to pick fights by <a href="http://www.merriam-webster.com/dictionary/nitpicking">nit-picking</a> other contributors. Asking for 
-      a source or reason (if not provided by the contributor in a contribution) is OK; otherwise you can always submit your own edit to fix 
+      a source or reason (if not provided by the contributor in a contribution) is OK; otherwise, you can always submit your edit to fix 
```

Comment:
I like how the "own" here emphasises that it’s the reader’s (own) edit being talked about, even if it is slightly pleonastic. No strong feeling towards keeping it if others find it looks much out of place.